src/Api/Api.php

```diff
@@ -58,17 +58,11 @@ public function getImageManager(): ImageManager
     /**
      * Set the manipulators.
      *
-     * @param ManipulatorInterface[] $manipulators Collection of manipulators.
+     * @param array $manipulators Collection of manipulators.
      */
```

Comment:
```suggestion
     * @throws \InvalidArgumentException
     */
```

Reply:
@ADmad good catch,  I added the missing phpdoc